### ISSUE TITLE
Improve spam score of netdata email notifications

### ISSUE THREAD

**Issue by arthurlogilab** [firehol/netdata#2587]
In some cases netdata notification emails have to go through some antispam (even though it is probably a good thing to have them only run in some internal network, or whitelist these kinds of emails, in some cases this is not possible). 

According to rspamd I get the following : 

```
X-Spamd-Result: default: False [5.00 / 30.00]
 MIME_HEADER_CTYPE_ONLY(2.00)
 FORGED_SENDER(0.30)
 MIME_HTML_ONLY(0.20)
 R_MISSING_CHARSET(2.50)
```

We can't do anything about FORGED_SENDER, but I guess the others could be fixed.

**Comment by Ferroin** [comment]
Other than the forged sender thing, most of those should be pretty easy to fix.

For MIME_HEADER_CTYPE_ONLY, we need to add Content-Transfer-Encoding and Content-Disposition headers (I think, but I'm not certain, that the correct values for these are 'UTF-8' and 'inline' respectively).

I'm not sure why MIME_HTML_ONLY is triggering, considering that a `text/plain` section is included.

For R_MISSING_CHARSET, I'm not 100% certain, but I think we just need to add `charset=` parameters to the Content-Type headers.

**Comment by arthurlogilab** [comment]
You can ignore the MIME_HTML_ONLY, this email is coming from a 1.6 version of netdata

**Comment by Ferroin** [comment]
Correction, Cont-Transfer-Encoding should be `quoted-printable` as far as I can tell.  Not sure about Content-Disposition, I've just looked through a handful of other e-mails in my inbox and don't see that on any of them.

The `charset` parameter for the MIME types may be tricky too, since the way we're sending the e-mail means we're dependent on the system locale settings to determine the character set.

**Comment by ktsaou** [comment]
merged #2588
